.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -30,6 +30,7 @@ Release notes are ready and available for all components | :red_circle: |   |
 Roadmap is up-to-date (information is available to create release highlights) | :red_circle: |   |
 Release ticket is cut, and there's a forum post announcing the start of the window | :red_circle: |   |
 [Any necessary security reviews](##Security-Reviews) are complete | :red_circle: |   |
+Each component release issue has an assigned owner | :red_circle: |   |
```

Comment:
Maybe put it at the top?
@dblock wondering if we need to get this reviewed from broader audience? I mean any changes to entry or exit criteria?

Reply:
Agreed we should put it on top.

Reply:
Sure Moved to the top, also @gaiksaya to your point, the created component release issues already had that field https://github.com/opensearch-project/job-scheduler/issues/662, but its just not prioritized.
<img width="324" alt="Screenshot 2024-08-16 at 11 38 52 AM" src="https://github.com/user-attachments/assets/61825b66-2c09-49d2-9e97-ecc0f64c8625">

Reply:
I’m going to approve this PR because I think it’s important, but would appreciate someone raises this on the #maintainers channel in slack for comments.